responsibleai/responsibleai/_internal/constants.py

```diff
@@ -7,90 +7,95 @@
 class ManagerNames(object):
     """Provide the manager names."""
 
-    CAUSAL = 'causal'
-    COUNTERFACTUAL = 'counterfactual'
-    ERROR_ANALYSIS = 'error_analysis'
-    EXPLAINER = 'explainer'
+    CAUSAL = "causal"
```

Comment:
strange that the single quotes are converted to double here, they should stay as single quotes

Reply:
Yeah, it's mainly because I'm using the `black` formatter which tries to convert single to double quotes where possible

I'll unconvert these for sure, but I'd highly recommend the `black` formatter for the Python code :)

https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html